walletlibrary/src/main/java/com/microsoft/walletlibrary/requests/requirements/constraints/VcTypeConstraint.kt

```diff
@@ -18,4 +19,9 @@ class VcTypeConstraint(internal val vcType: String): VerifiedIdConstraint {
             return false
         return verifiedId.types.contains(vcType)
     }
+
+    override fun matches(verifiedId: VerifiedId) {
+        if (!doesMatch(verifiedId))
+            throw VerifiedIdTypeIsNotRequestedTypeException("Provided Verified Id type does not match requested type.")
```

Comment:
Should we surface up which type is requested that the Verified ID does not have here?

Reply:
Added them.